recipes/cairocffi/dlopen-patch.linux.txt

```diff
@@ -0,0 +1,11 @@
+--- cairocffi/__init__.py	2016-11-16 13:45:03.606537277 +0100
++++ cairocffi/__init__.py.linux	2016-11-16 13:45:38.591975235 +0100
+@@ -43,7 +43,7 @@
+     raise OSError("dlopen() failed to load a library: %s" % ' / '.join(names))
+ 
+ 
+-cairo = dlopen(ffi, 'cairo', 'cairo-2')
++cairo = ffi.dlopen('libcairo.so')
```

Comment:
I don't think this is necessary. First, ffi.dlopen does the same as the system call. That means it will use the RPATH of Python, which already points to the environment prefix, right?
Second, we don't want to restrict the user too much. Maybe he wants to set LD_LIBRARY_PATH such that a different cairo is used for some reason.